sources/targets/Stride.Core.props

```
@@ -44,8 +44,8 @@
     Settings StridePlatform specific
   -->
   <PropertyGroup>
-    <!-- Using C# version 10.0 which targets Net6.0 -->
-    <LangVersion>10.0</LangVersion>
+    <!-- Using C# version 11.0 which targets net8.0 -->
+    <LangVersion>11.0</LangVersion>
```

We could use `latest` instead so that we are always on the latest with regards to the compiler capabilities.
```suggestion
    <LangVersion>latest</LangVersion>
```

---

I just removed since it should use latest available from the compiler by default.